src/workerd/api/crypto/crc-impl.h

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) 2017-2025 Cloudflare, Inc.
+// Licensed under the Apache 2.0 license found in the LICENSE file or at:
+//     https://opensource.org/licenses/Apache-2.0
+
+#pragma once
+#include <stdint.h>
+
+uint32_t crc32c(uint32_t crc, const uint8_t *data, unsigned int length);
+uint64_t crc64nvme(uint64_t crc, const uint8_t *data, unsigned int length);
```

Comment:
Nit: code comments explaining the differences between these two would be helpful. The `nvme` is not going to be that meaningful to new folks looking at this.